opentracing/README.md

```diff
@@ -0,0 +1,109 @@
+# DCP RFC
```

Comment:
be sure to read this README (preferably via the "View" button that honors the formatting).